legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/api/AbstractGitLabApiTest.java

```diff
@@ -42,6 +42,13 @@
  * Prepares subclass GitLab integration tests.
  * Only run before the GitLab integration tests during integration-test phase in-between docker start and stop.
  * Skipped during Junit tests.
+ *
+ * IMPORTANT: Please note that if wishing to run this integration test suite locally,
+ * it is requiresd to have local Docker installed, with recommended at least 4 Gi memory allocated.
+ * If encountering GitLab not responding issues, please raise the wait time to approximately 450000ms on line 118,
+ * legend-sdlc-server/pom.xml.
+ * If your machine cannot suffice the resource requirements or fail to pass the test suite, please use maven profile
+ * skip-integration-tests to skip them during the build.
  */
```

Comment:
This is a very useful comment. Some suggestions. 

1/ Minor typo in "requiresd"
2/ We should say that this only applies to environments where the Docker container is run in a VM.
3/ The reference to line 118 is going to quickly go out of date as pom.xml evolves. Maybe just refer to the pom.xml plugin/property by name ?

Reply:
Roger! Thanks